scripts/e2e-v3.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+export TEST_SUITE_TYPE="jsunit"
+export TEST_RESULT_FILE_DIR=${REPO}/src/v3/build2/reports/e2e
+echo $TEST_SUITE_TYPE > $TEST_SUITE_TYPE_FILE
+echo $TEST_RESULT_FILE_DIR > $TEST_RESULT_FILE_DIR_FILE
+
+source ${OKTA_HOME}/${REPO}/scripts/setup.sh
+
+cd ${OKTA_HOME}/${REPO}
+
+setup_service google-chrome-stable 91.0.4472.77-1
```

Comment:
note: v2 uses `setup_service google-chrome-stable 83.0.4103.61-1`

Reply:
Yea, also noticed that, I feel it's most probably not get updated. Also feel using newer version might be better choice, since it exposes latest browser changes to tests, like cookie related changes. I think we can discuss more on this in office hour next week.